### PR TITLE
fix mobile alignment issues

### PR DESCRIPTION
**Pull request by Jibz1**
Before
Screen Shot 2022-07-28 at 10 39 46 PM

After
Screen Shot 2022-07-28 at 10 39 12 PM

### CODE REVIEW

**Review by Andrew47 — CHANGES_REQUESTED** [review]
Great that you have spotted the alignment issues in mobile. The suggested improvements are much better than before.

The text is still not quite alligned as shown when adding a horizontal red line (see 'Before' in diagram below). After some CSS experimentation, the way forward that leads to perfect alignment seems to lie in the line heights (see 'After' in diagram below).
image

The issue is that the line heights do not match in the left and right columns. By setting `text-sm` in the right column, this gives it a font-size of 0.875rem which matches well with the Figma specification of 14px but it also gives a line height of 1.25rem. The Figama Design specifies a line-height of 13px. Hence, one way forward would be to remove `text-sm` from `.itemviewcontentvalue` (implemented in the above diagram).

File: explorer/client/src/pages/transaction-result/TransactionResult.module.css [hunk 1]
```diff
@@ -47,8 +47,9 @@
 }
 
 .itemviewcontentvalue {
-    @apply text-sm pt-2 pb-2 text-gray-800 contents md:block;
```
Comment (Andrew47) [inline explorer/client/src/pages/transaction-result/TransactionResult.module.css:50]: Remove `pt-2` and `pb-2` here.

**Comment by Jibz1** [comment]
@Andrew47 thanks for the feedback. Please review.

**Comment by Andrew47** [comment]
> @Andrew47 thanks for the feedback. Please review.

@Jibz1 Apologies for the confusion. Looks like `text-sm` also needs to be added to `.itemviewcontentlabel`.

**Comment by stella3d** [comment]
looks like the tiny mobile sizes are now aligned right - this is iphone SE size
image

**Review by stella3d — APPROVED** [review]

**Pull request merged**